### ISSUE TITLE
An error occurred mounting one of your file systems when running any rancher commands.

### ISSUE THREAD

**Issue by veleek** [rancher-sandbox/rancher-desktop#1096]
### Rancher Desktop Version

0.6.1

### Rancher Desktop K8s Version

1.21.7

### What operating system are you using?

Windows

### Operating System / Build Version

Windows 11 V 21H2 (OS Build 22000.318)

### What CPU architecture are you using?

x64

### Windows User Only

N/A

### Actual Behavior

Running any commands like `nerdctl` or launching into the `rancher-desktop` WSL container always shows an error as the first line:

```
An error occurred mounting one of your file systems. Please run 'dmesg' for more details.
```

### Steps to Reproduce

Run any rancher desktop command that calls into the WSL container.

### Result

Digging through `dmesg` shows the following errors:

```
[    3.484280] init: (1) ERROR: ConfigApplyWindowsLibPath:2474: open /etc/ld.so.conf.d/ld.wsl.conf
[    3.484282]  failed 2
[    3.488955] init: (1) ERROR: UtilCreateProcessAndWait:673: /bin/mount failed with status 0x
[    3.488956] 100
[    3.488960] init: (1) ERROR: ConfigMountFsTab:2529: Processing fstab with mount -a failed.
```

The file `/etc/sd.so.conf.d/ld.wsl.conf` doesn't exist so this could be the root of the problem.

### Expected Behavior

No errors should occur.  Currently it's not obvious whether this actually impacts the app.

**Comment by veleek** [comment]
@mook-as - Nope, no custom `/etc/fstab`, no `fstab` at all!!  Not sure if it's expected to be there but I haven't done anything custom with the distro.

Making an empty `fstab` file and then restarting everything (killing the WSL distro) seems to make the error go away.  

So, this _may_ just be a WSL error or some change based on a newer version of WSL/Windows, or it could be specific to the WSL config for the rancher distro?

------

Also, I'm not sure if it was clear, but I can still launch the distro successfully and run `nerdctl` commands and start/stop containers even with it giving those errors.

```
PS> wsl --list
Windows Subsystem for Linux Distributions:
Ubuntu (Default)
rancher-desktop
rancher-desktop-data
PS> wsl -d rancher-desktop
An error occurred mounting one of your file systems. Please run 'dmesg' for more details.
/mnt/c/Users/veleek # ls /etc
alpine-release        ca-certificates.conf  hosts                 logrotate.d           os-release            rancher               shadow
apk                   ethertypes            iptables              machine-id            passwd                resolv.conf           ssl
ca-certificates       hostname              issue                 mtab                  pki                   securetty             udhcpd.conf
/mnt/c/Users/veleek # exit
PS> nerdctl ps
An error occurred mounting one of your file systems. Please run 'dmesg' for more details.
CONTAINER ID    IMAGE                                      COMMAND                   CREATED         STATUS    PORTS                     NAMES
2ff673a9274b    docker.io/maptiler/tileserver-gl:latest    "/app/docker-entrypo…"    44 hours ago    Up        0.0.0.0:8080->8080/tcp
```

**Comment by mook-as** [comment]
Hmm, okay, so we just need a dummy `fstab` to exist? Then this _should_ be fixed in 0.7, since we added `mountFsTab = false` to `wsl.conf`.   (For tracking purposes when I look at this issue again: this landed in wsl-distro [0.6](https://github.com/rancher-sandbox/rancher-desktop-wsl-distro/commit/1df971f8bc3d2e5c537be90dd31beabce0b6941a), which landed [after](https://github.com/rancher-sandbox/rancher-desktop/commit/70952ed6e8ea43e961926d15879d068a25c98ca6) Rancher Desktop 0.6.)

If you'd like, you can try adding the line to `/etc/wsl.conf` in the `rancher-desktop` WSL2 distribution to confirm that the error message went away.  You can just drop the [whole file](https://raw.githubusercontent.com/rancher-sandbox/rancher-desktop-wsl-distro/v0.9/files/wsl.conf) there — you don't have one yet.

**Comment by veleek** [comment]
Seems to have worked.  I removed `fstab` and restarted everything and no errors!  Thanks for the help.   Glad it's going away already.
